akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala

```diff
@@ -82,11 +82,19 @@ class InteractionPatternsSpec extends ScalaTestWithActorTestKit with AnyWordSpec
       val context = new {
         def self = probe.ref
       }
+
       // #request-response-send
       cookieFabric ! CookieFabric.Request("give me cookies", context.self)
       // #request-response-send
 
       probe.receiveMessage()
+
+      Behaviors.setup { context =>
+        // #ignore-reply
+        cookieFabric ! CookieFabric.Request("don't send cookies back", context.system.ignoreRef)
+        // #ignore-reply
+        Behaviors.empty[Nothing]
+      }
```

Comment:
I'm not adding any new test. No probe since any message is ignore anyway and this has already been tested elsewhere.